ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-settings/source-selector.tsx

```diff
@@ -293,34 +294,36 @@ const SourceSelector = ({ search }: Props) => {
           <Grid container alignItems="stretch" direction="row" wrap="nowrap">
             <Grid container direction="row" alignItems="center">
               <Grid item className="pr-2">
-                <CheckIcon
-                  className={
-                    shouldBeSelected({ srcId: 'all', sources })
-                      ? ''
-                      : 'invisible'
-                  }
-                />
+                {shouldBeSelected({ srcId: 'all', sources }) ? (
+                  <CheckBoxIcon />
+                ) : (
+                  <CheckBoxOutlineBlankIcon />
+                )}
```

Comment:
✏️ Not a big deal, but we could factor this out as a component that just displace the proper icon.

Reply:
Will do this if we need the functionality in another component.